server/src/main/java/org/apache/iotdb/db/query/dataset/NewEngineDataSetWithoutValueFilter.java

```diff
@@ -86,7 +88,7 @@ public void run() {
           reader.setManagedByQueryManager(false);
         }
       } catch (IOException | InterruptedException e) {
-        e.printStackTrace();
+        LOGGER.error(e.getMessage());
```

Comment:
Log the exception instead of merely its message unless the exception is very tiny otherwise, it may be a disaster in future debugs.